docs/source/recipes/images.md

```diff
@@ -0,0 +1,51 @@
+# Images
```

Comment:
All docs need HTML meta tags for SEO, and a label for creating links from cross-references.

```suggestion
---
myst:
  html_meta:
    "description": "Volto code recipe for images"
    "property=og:description": "Volto code recipe for images"
    "property=og:title": "Volto code recipe for images"
    "keywords": "Plone, Volto, code, recipe, images"
---

(images-label)=

# Images
```